doc/source/getting_started/faq.rst

```diff
@@ -12,6 +12,28 @@ Design Language (APDL), Ansys Fluent, and other Ansys products.
 You can use PyAnsys libraries within a Python environment of your choice
 in conjunction with external Python libraries.
 
+What Ansys license do I need to run the Geometry service?
+---------------------------------------------------------
+
+.. note::
+
+   Answered in https://github.com/ansys/pyansys-geometry/discussions/754
+
+The Ansys Geometry service is a headless service developed on top of the
+modelling libraries underneath Discovery and SpaceClaim.
+
+Both in its standalone and dockerized versions, the Ansys Geometry Service
+requires a **Discovery Modeling** license to run.
+
+In order to run PyAnsys Geometry against other backends, such as Discovery
+or SpaceClaim, users will need an Ansys license that allows them to run these
+Ansys products.
+
+The **Discovery Modeling** license is one of these licenses, but there are others
+such as the Ansys Mechanical Enterprise license that would also allow users to run
+these Ansys products. However, the Geometry service is not compatible with any other
+license.
```

Comment:
```suggestion
The **Discovery Modeling** license is one of these licenses, but there are others,
such as the Ansys Mechanical Enterprise license, that also allow users to run
these Ansys products. However, the Geometry service is not compatible with any other
license.
```
Should "Ansys Mechanical Enterprise** be bolded as a proper license name--like **Discovery Modeling** licesnse?

Reply:
I'd avoid making it bold. My intention is that users that read diagonbally would at least catch that only the **Discovery Modeling** license is compatible with the Ansys Geometry service - which is the title of the question. It was done on purpose =)